cmd/tester/main.go

```diff
@@ -63,17 +64,32 @@ type output struct {
 }
 
 func main() {
-	cipFilePath := flag.String("policy", "", "path to ClusterImagePolicy")
+	cipFilePath := flag.String("policy", "", "path to ClusterImagePolicy or URL to fetch from (http/https)")
 	image := flag.String("image", "", "image to compare against policy")
 	flag.Parse()
 	if *cipFilePath == "" || *image == "" {
 		flag.Usage()
 		os.Exit(1)
 	}
 
-	cipRaw, err := ioutil.ReadFile(*cipFilePath)
-	if err != nil {
-		log.Fatal(err)
+	var cipRaw []byte
+	var err error
+	if strings.HasPrefix(*cipFilePath, "https://") || strings.HasPrefix(*cipFilePath, "http://") {
+		log.Printf("Fetching CIP from: %s", *cipFilePath)
+		resp, err := http.Get(*cipFilePath)
+		if err != nil {
+			log.Fatal(err)
+		}
+		defer resp.Body.Close()
```

Comment:
You could move this code to a function so the defer statements get executed, and then log a Fatal error here.